Source/player.cpp

```diff
@@ -3285,8 +3276,8 @@ void StartWarpLvl(int pnum, int pidx)
 
 void ProcessPlayers()
 {
-	if ((DWORD)MyPlayerId >= MAX_PLRS) {
-		app_fatal("ProcessPlayers: illegal player %i", MyPlayerId);
+	if (MyPlayerId >= MAX_PLRS) {

```

Comment:
Why not compare it to `Players.size()`?

Reply:
Yeah this should be `Players.size()`

Reply:
Ok looks like there is still quite a bit of that around. I'll try to update that later today.